tests/middleware/test_trace_logging.py

```diff
@@ -13,10 +13,10 @@
     "disable_existing_loggers": False,
     "formatters": {
         "test_formatter_default": {
-            "format": "[TEST_DEFAULT] %(levelname)-9s %(name)s - %(lineno)d - %(message)s"
+            "format": "[TEST_DEFAULT] %(levelname)-9s %(name)s - %(lineno)d - %(message)s"  # noqa: E501
```

Comment:
Thoughts on ignoring E501 (line too long) by default, ie putting it in setup.cfg? Something I've been thinking of doing in HTTPCore and HTTPX too...

Reply:
I'd rather not have it by default so that we are explicit about tolerating it on some rare occasions where the line is long enough but it wouldnt make sense to split it, but I guess it's really a matter of taste and I'm happy to put it by default

Reply:
Sure, not sure either, so let's keep things like they are for now. :)